docs/cert-manager.md

```diff
@@ -307,7 +307,7 @@ minio.<namespace>
 minio.<namespace>.svc
 minio.<namespace>.svc.<cluster domain>
 *.<tenant-name>-hl.<namespace>.svc.<cluster domain>
-*.<namespace>.svc.<cluster domain>
+*.minio.<namespace>.svc.<cluster domain>
```

Comment:
@pjuarezd @allanrogerr It's a good idea to remove `*.<namespace>.svc.<cluster domain>` from the certificate. But why add `*.minio.<namespace>.svc.<cluster domain>`? The `minio` service is not a headless service, so it will always be addressed using `minio.<namespace>.svc.<cluster domain>` (already in the certificate). Only headless services need a wildcard domain...

PS: I think `*.<tenant-name>.minio.<namespace>.svc.<cluster domain>` on line 311 is obsolete too. You can't have services that have this amount of dots before the `<namespace>.svc.<cluster domain>`...